### PR TITLE
Add stubs for nf-core/raredisease modules

### PR DESCRIPTION
**Pull request by ryanjameskennedy**
Added stubs for modules that will be used for nf-core/raredisease
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by muffato — COMMENTED** [review]
It seems this needs to be defined, and it should address one of the failing tests

File: modules/nf-core/gatk4/collectreadcounts/main.nf [hunk 1]
```diff
@@ -50,4 +50,18 @@ process GATK4_COLLECTREADCOUNTS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+
+    stub:
```
[the remark this answers is not in the document]
Comment (ryanjameskennedy) [reply modules/nf-core/gatk4/collectreadcounts/main.nf:54]: Yeah it doesn't work without the args line!

**Review by muffato — APPROVED** [review]

**Comment by muffato** [comment]
Hi @ryanjameskennedy . I'm happy with the change. I can't merge it because the branch needs to be up-to-date with master. You should have a button to update your branch. Could you please hit that ? Then I'll be able to merge this PR

(assuming no one else has pushed anything to master in the meantime ! otherwise we'll have to keep trying that again)

**Review by jemten — APPROVED** [review]
LGTM

**Pull request merged**